website/content/docs/agent/caching/persistent-caches/index.mdx

```diff
@@ -1,9 +1,49 @@
 ---
 layout: docs
-page_title: Vault Agent Persistent Cache Types
-description: Persistent Cache Types for Vault Agent Caching
+page_title: Vault Agent Persistent Caching
+description: Vault Agent Caching
 ---
 
-# Vault Agent Persistent Cache Types
+# Vault Agent Persistent Caching
+
+Vault Agent can restore tokens and leases from a persistent cache file created
+by a previous Vault Agent process. The persistent cache is a BoltDB file that
+includes tuples encrypted by a generated encryption key. The encrypted tuples
+include the Vault token used to retrieve secrets, leases for tokens/secrets, and
+secret values.
+
+-> **Note:** Vault Agent Persistent Caching will only restore _leased_
+secrets. Secrets that are not renewable, such as KV v2, will not be persisted.
+
+In order to use Vault Agent persistent cache, auto-auth must be used. During the
+restoration of the cache, Vault Agent will pre-populate auto-auth with the
+persisted token. This token is required to renew restored leases. If the token
+has expired, the cached leases will be evicted and secrets will need to be
+retrieved from Vault.
```

Comment:
This paragraph goes quite into depth on the implementation details, but I think the user only needs to know the auto-auth requirement and the behaviour when the token's expired. The suggestion below is a crude attempt at simplifying it down to those points.

```suggestion
In order to use Vault Agent persistent cache, auto-auth must be used. If the
auto-auth token has expired by the time the cache is restored, the cache will
be invalidated and secrets will need to be re-fetched from Vault.
```

Reply:
I like this, updated!